src/composables/transactionTx.ts

```diff
@@ -137,6 +139,7 @@ export function useTransactionTx({
     isDex,
     isMultisig,
     direction,
+    multisigType,
```

Comment:
This is not used anywhere and also the name makes no sense - please see my previous comment.